content/docs/command-reference/plots/diff.md

```diff
@@ -47,14 +47,17 @@ please see `dvc plots`.
 
 - `--targets <path>` - specific metric files to visualize. These must be listed
   in a [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) file
-  (see the `--plots` option of `dvc run`). When specifying multiple `--targets`
-  and `revisions`, you may use `--` after this option's arguments, e.g.:
+  (see the `--plots` option of `dvc run`). When specifying arguments for
+  `--targets` before `revisions`, you may use `--` after this option's arguments
+  , e.g.:
 
   ```dvc
   $ dvc plots diff --targets t1.json t2.csv -- HEAD v1 v2
   ```
 
-  Alternatively, you can also run above statement as:
+  Without `--`, this statement will show an error as argument parser confuses
+  `revisions` as arguments for `--targets` option. Alternatively, you can also
```

Comment:
In https://github.com/iterative/dvc.org/pull/1567#pullrequestreview-447399307, I guess you asked me whether to keeps this part on not.  This will be helpful in case a user is still not sure why `--` has to be used.

Reply:
Yeah but we also want this to be a short note. Probably no need to exact explanation as to why. People don't know what a parser is anyway.